index.js

```diff
@@ -0,0 +1,91 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let className = document.querySelector("#className");
+  let teacherName = document.querySelector("#teacherName");
+  let addClass = document.querySelector("#addClass");
+  let enrollClass = document.querySelector("#studentClass");
+  let enrollName = document.querySelector("#studentName");
+  let enrollAge = document.querySelector("#studentAge");
+  let enrollCity = document.querySelector("#studentCity");
+  let enrollGrade = document.querySelector("#studentGrade");
+  let classList = document.querySelector("#classList");
+  let cityList = document.querySelector("#cityList");
+  let checkFailing = document.querySelector("#checkFailing");
+  let listSubmit = document.querySelector("#listSubmit");
+  let form1 = document.querySelector("#one");
+  let form2 = document.querySelector("#two");
+  let form3 = document.querySelector("#three");
+  let div1 = document.querySelector("#divOne");
+  let div2 = document.querySelector("#divTwo");
+  let div3 = document.querySelector("#three");
+
+  form1.addEventListener("submit", async e => {
+    e.preventDefault();
+    try {
+      await axios.post(`http://localhost:3000/class/${className.value}/${teacherName.value}`).then(res => {
+          let result = document.createElement("p");
+          let newClass =
+            className.value.charAt(0).toUpperCase() + className.value.slice(1);
+          let teacher =
+            teacherName.value.charAt(0).toUpperCase() +
+            teacherName.value.slice(1);
+          result.innerText = `Teacher: ${teacher} : Class: ${newClass}`;
+          div1.appendChild(result);
+          debugger;
+          console.log(res.data.message);
+        });
+    } catch (err) {
+      console.log(err);
+    }
+  });
+
+  form2.addEventListener("submit", async e => {
+    e.preventDefault();
+    try {
+      await axios.post(`http://localhost:3000/class/${enrollClass.value}/enroll`, {
```

Comment:
You can just assign the return value of this axios request to a variable `res` since you don't have to do `.then` when you use axios with async-await